### DISCUSSION TITLE
How to make all nodes visibble?

### DISCUSSION THREAD

**Discussion by deltabert** [zachowj/node-red-contrib-home-assistant-websocket#624] [Q&A] [answered]
Hi all,
I have a problem with the ode-red-contrib-home-assistant nodes.
I have two platforms; an Intel NUC and a RPi4, both with node-red in the latest version.

I installed the node-red-contrib-home-assistant-websocket on both machines, but the nodes are different on the machines.
On the NUC i can have all 22 nodes available while on the RPi4 in have only 8 of the 22 nodes usable; see pictures

Does anyone has an idea what causes this?  On both machines i installed it via the UI :: Manage palettes.
 
![image](https://user-images.githubusercontent.com/5196699/161385350-b05d89c1-d787-4a8a-a9d2-6b2218a82255.png)

![image](https://user-images.githubusercontent.com/5196699/161385366-458d5626-94fd-4dab-8a69-720328335818.png)

Kind regards, Bert

**Comment by zachowj** [comment, marked as answer]
Remove node-red-contrib-home-assistant from the machine in the top image and reboot node-red

**Reply by deltabert** [reply]
Thank you @zachowj Jason,
I deleted it and from the file system and installed it again i the CLI. All is fine now. I could not delete it from the UI.
Thanks, and regards, Bert
